paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -53,11 +53,17 @@ internal data class BuyButtonState(
 
 internal sealed interface PaymentSheetScreen {
 
+    enum class AnimationStyle {
+        PrimaryButtonAnchored,
+        FullPage,
+    }
```

Comment:
if I add a new screen in the future, how should I decide which animation style is right? Should there be a default option and we just set an override (e.g. PrimaryButtonAnchored is specified for the two screens that need it and we otherwise don't set this)? Or could you add a comment or something else to make this clear?

Reply:
Yeah, I can leave some comments as to what they do, as well as set a default.

Reply:
ty!